### ISSUE TITLE
prod env cache problem

### ISSUE THREAD

**Issue by alexbumbacea** [goaop/goaop-symfony-bundle#11]
Hi,

I have a problem in production, it check for write permission on "cache/aspect" folder. And it doesn't have the right... it fails. 
This shouldn't happen as the entire "cache" should be written by "cache:warmup".

Steps to reproduce:
1. Run as root php ./bin/console cache:clear -e=prod
2. Run as www-data any other command (make sure you still do -e=prod )

**Comment by lisachenko** [comment]
Hi! It's always bad idea to run any symfony command from the root user. You should always use the same user name as for your web server.

Alternatively, you could configure cache file mode and check umask settings to enable creation of directories with write/read rights for everyone (not so good idea, but it's also a solution)

**Comment by alexbumbacea** [comment]
Hi 
Right about the root vs web server user stuff. :). 
If you look at how symfony handles "cache:warmup" in prod env, this should work. All the write work should be done during the warmup.

**Comment by alexbumbacea** [comment]
Thanks for the advice. The problem was already resolved on my "prod" environment when I raised the issue.
But I still think that the permission check should not be done in those conditions (as it has nothing to write)

**Comment by lisachenko** [comment]
If I remember correctly, for the production mode there is no filemode check. But I need to look this or check... If you seeing the mistake and can fix it by PR, then just send me it, I will merge it after review.

**Comment by alexbumbacea** [comment]
goaop/framework/src/Instrument/ClassLoading/CachePathManager.php:85

**Comment by lisachenko** [comment]
I'm from the phone. Definitely something is wrong, but cache directory should be present, isn't it? (And in this case we won't try to check if our directory it's writable or not)

**Comment by alexbumbacea** [comment]
I think i copied that from a tag.
`if (!$this->kernel->hasFeature(Features::PREBUILT_CACHE) && !is_writable($this->cacheDir))`
This is the section i'm referring too.

**Comment by lisachenko** [comment]
Yes, now I can see your case. But I hadn't time right now to fix this...
